duckduckgo/macos-browser
# Rename preferences to settings

Task/Issue URL: https://app.asana.com/0/1177771139624306/1202096826651508/f
Tech Design URL:
CC:

**Description**:

This PR renames "Preferences" to "Settings" everywhere **except** the menu bar's Preferences option. We agreed that that entry should remain "Preferences" for the sake of consistency with the platform.

**Steps to test this PR**:
1. Open the app and check that the menu bar's Preferences option hasn't changed
1. Open the More Options ... menu and check that "Settings" is used, and that the icons are correct in the menu
1. Open settings itself and check that the title of the tab and page switcher button are both "Settings"
1. Verify that there are no other user-facing "Preferences" strings to be changed

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
